{"0.0.0.dev0":{"info":{"author":null,"author_email":null,"bugtrack_url":null,"classifiers":[],"description_content_type":null,"docs_url":null,"download_url":null,"downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":null,"keywords":null,"license":null,"maintainer":null,"maintainer_email":null,"name":"ExceptionMan","package_url":"https://pypi.org/project/ExceptionMan/","platform":null,"project_url":"https://pypi.org/project/ExceptionMan/","project_urls":null,"provides_extra":null,"release_url":"https://pypi.org/project/ExceptionMan/0.0.0.dev0/","requires_dist":["markitup","ansi-sgr"],"requires_python":">=3.10","summary":null,"version":"0.0.0.dev0","yanked":false,"yanked_reason":null},"last_serial":25913582,"urls":[{"comment_text":"","digests":{"blake2b_256":"766b57b90a54185f4b3843d43be44702a2879a20a779d57266e286ed8ad313d3","md5":"6622cfbef253a025139f378ab39596b3","sha256":"9d8ed810b086105891571c49fb57d65c4a4fd0d2610b78d96726caa45a1e4fea"},"downloads":-1,"filename":"ExceptionMan-0.0.0.dev0-py3-none-any.whl","has_sig":false,"md5_digest":"6622cfbef253a025139f378ab39596b3","packagetype":"bdist_wheel","python_version":"py3","requires_python":">=3.10","size":2375,"upload_time":"2024-08-26T10:40:04","upload_time_iso_8601":"2024-08-26T10:40:04.968744Z","url":"https://files.pythonhosted.org/packages/76/6b/57b90a54185f4b3843d43be44702a2879a20a779d57266e286ed8ad313d3/ExceptionMan-0.0.0.dev0-py3-none-any.whl","yanked":false,"yanked_reason":null},{"comment_text":"","digests":{"blake2b_256":"d04d2720183451bd048779b6b954ed996059d5374e2375d4f15811acf8cb8827","md5":"ecc949ea92f74af75fee319ada66d0c7","sha256":"72ef57d6b1d9c6e0a563485b25c11851b18b586492ebfd64a5e1d393ed6fcb6b"},"downloads":-1,"filename":"exceptionman-0.0.0.dev0.tar.gz","has_sig":false,"md5_digest":"ecc949ea92f74af75fee319ada66d0c7","packagetype":"sdist","python_version":"source","requires_python":">=3.10","size":2110,"upload_time":"2024-08-26T10:40:06","upload_time_iso_8601":"2024-08-26T10:40:06.138519Z","url":"https://files.pythonhosted.org/packages/d0/4d/2720183451bd048779b6b954ed996059d5374e2375d4f15811acf8cb8827/exceptionman-0.0.0.dev0.tar.gz","yanked":false,"yanked_reason":null}],"vulnerabilities":[]},"0.0.0.dev1":{"info":{"author":null,"author_email":null,"bugtrack_url":null,"classifiers":[],"description_content_type":null,"docs_url":null,"download_url":null,"downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":null,"keywords":null,"license":null,"maintainer":null,"maintainer_email":null,"name":"ExceptionMan","package_url":"https://pypi.org/project/ExceptionMan/","platform":null,"project_url":"https://pypi.org/project/ExceptionMan/","project_urls":null,"provides_extra":null,"release_url":"https://pypi.org/project/ExceptionMan/0.0.0.dev1/","requires_dist":["markitup","ansi-sgr"],"requires_python":">=3.10","summary":null,"version":"0.0.0.dev1","yanked":false,"yanked_reason":null},"last_serial":25913582,"urls":[{"comment_text":"","digests":{"blake2b_256":"44a499b0cc233442f8a0cc03b425d83f9f8461f7f71683f0f0e60782d4422a6d","md5":"f54c48aad3b058b7c16a0cc07f998954","sha256":"db5bdc89dd4e48647af2c8725414c727b9ff396b0bc745c8b183a993c87559a1"},"downloads":-1,"filename":"ExceptionMan-0.0.0.dev1-py3-none-any.whl","has_sig":false,"md5_digest":"f54c48aad3b058b7c16a0cc07f998954","packagetype":"bdist_wheel","python_version":"py3","requires_python":">=3.10","size":2441,"upload_time":"2024-08-27T13:27:38","upload_time_iso_8601":"2024-08-27T13:27:38.470990Z","url":"https://files.pythonhosted.org/packages/44/a4/99b0cc233442f8a0cc03b425d83f9f8461f7f71683f0f0e60782d4422a6d/ExceptionMan-0.0.0.dev1-py3-none-any.whl","yanked":false,"yanked_reason":null},{"comment_text":"","digests":{"blake2b_256":"2ea1ddf7cf5a558a02f72c7a764cc2ac5acb78aad6f187cc3f99aa71e0835c62","md5":"d2035bc3dc2ca3641f843c877244b96a","sha256":"bdf1c6286ccf7a45c5e21fe4275f1cdbffc4a2569a3b490c13bb6dd75e9f68b3"},"downloads":-1,"filename":"exceptionman-0.0.0.dev1.tar.gz","has_sig":false,"md5_digest":"d2035bc3dc2ca3641f843c877244b96a","packagetype":"sdist","python_version":"source","requires_python":">=3.10","size":2146,"upload_time":"2024-08-27T13:27:39","upload_time_iso_8601":"2024-08-27T13:27:39.615445Z","url":"https://files.pythonhosted.org/packages/2e/a1/ddf7cf5a558a02f72c7a764cc2ac5acb78aad6f187cc3f99aa71e0835c62/exceptionman-0.0.0.dev1.tar.gz","yanked":false,"yanked_reason":null}],"vulnerabilities":[]},"0.0.0.dev10":{"info":{"author":null,"author_email":null,"bugtrack_url":null,"classifiers":[],"description_content_type":null,"docs_url":null,"download_url":null,"downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":null,"keywords":null,"license":null,"maintainer":null,"maintainer_email":null,"name":"ExceptionMan","package_url":"https://pypi.org/project/ExceptionMan/","platform":null,"project_url":"https://pypi.org/project/ExceptionMan/","project_urls":null,"provides_extra":null,"release_url":"https://pypi.org/project/ExceptionMan/0.0.0.dev10/","requires_dist":["MDit==0.0.0.dev11","rich>=13.5","pygments"],"requires_python":">=3.10","summary":null,"version":"0.0.0.dev10","yanked":false,"yanked_reason":null},"last_serial":25913582,"urls":[{"comment_text":"","digests":{"blake2b_256":"f7512dd7485e537989aa498239afbd580e0db17c23f5b4b6924fb7cff633bff7","md5":"8c2809da893fd6d4c4cc7186bbe218cd","sha256":"117fedcb6f80da299ffccbc0ec86de99a558ecda4199ea335bb97017c00c3c0c"},"downloads":-1,"filename":"ExceptionMan-0.0.0.dev10-py3-none-any.whl","has_sig":false,"md5_digest":"8c2809da893fd6d4c4cc7186bbe218cd","packagetype":"bdist_wheel","python_version":"py3","requires_python":">=3.10","size":9042,"upload_time":"2024-09-24T10:48:09","upload_time_iso_8601":"2024-09-24T10:48:09.872191Z","url":"https://files.pythonhosted.org/packages/f7/51/2dd7485e537989aa498239afbd580e0db17c23f5b4b6924fb7cff633bff7/ExceptionMan-0.0.0.dev10-py3-none-any.whl","yanked":false,"yanked_reason":null},{"comment_text":"","digests":{"blake2b_256":"02404b46d7174ba612310bb4ef35cbb7ad2016c237521efdad8ff96627677725","md5":"b208210716cefdf4f39cbf751ee9ca4e","sha256":"8301e7a62280cf4620fc3e4b2b894c1a75972523b5c4da10d063e2c20210cb85"},"downloads":-1,"filename":"exceptionman-0.0.0.dev10.tar.gz","has_sig":false,"md5_digest":"b208210716cefdf4f39cbf751ee9ca4e","packagetype":"sdist","python_version":"source","requires_python":">=3.10","size":8160,"upload_time":"2024-09-24T10:48:10","upload_time_iso_8601":"2024-09-24T10:48:10.971863Z","url":"https://files.pythonhosted.org/packages/02/40/4b46d7174ba612310bb4ef35cbb7ad2016c237521efdad8ff96627677725/exceptionman-0.0.0.dev10.tar.gz","yanked":false,"yanked_reason":null}],"vulnerabilities":[]},"0.0.0.dev11":{"info":{"author":null,"author_email":null,"bugtrack_url":null,"classifiers":[],"description_content_type":null,"docs_url":null,"download_url":null,"downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":null,"keywords":null,"license":null,"maintainer":null,"maintainer_email":null,"name":"ExceptionMan","package_url":"https://pypi.org/project/ExceptionMan/","platform":null,"project_url":"https://pypi.org/project/ExceptionMan/","project_urls":null,"provides_extra":null,"release_url":"https://pypi.org/project/ExceptionMan/0.0.0.dev11/","requires_dist":["MDit==0.0.0.dev11","rich>=13.5","pygments"],"requires_python":">=3.10","summary":null,"version":"0.0.0.dev11","yanked":false,"yanked_reason":null},"last_serial":25913582,"urls":[{"comment_text":"","digests":{"blake2b_256":"6c7163f11fdfe33d2fc3fb61a98582a37a6e845a6ff6bcc2e9070b8cf8f0ea81","md5":"183af1b05ac3efb578e78f78cc72943a","sha256":"d3d3cff617267621ec873dc2c875de4a55cff1b38fbaa51933aca21d8c2e106a"},"downloads":-1,"filename":"ExceptionMan-0.0.0.dev11-py3-none-any.whl","has_sig":false,"md5_digest":"183af1b05ac3efb578e78f78cc72943a","packagetype":"bdist_wheel","python_version":"py3","requires_python":">=3.10","size":9042,"upload_time":"2024-09-24T10:54:34","upload_time_iso_8601":"2024-09-24T10:54:34.555324Z","url":"https://files.pythonhosted.org/packages/6c/71/63f11fdfe33d2fc3fb61a98582a37a6e845a6ff6bcc2e9070b8cf8f0ea81/ExceptionMan-0.0.0.dev11-py3-none-any.whl","yanked":false,"yanked_reason":null},{"comment_text":"","digests":{"blake2b_256":"8e0393a77bd7662d46da63dcfbefeeb4470d9424d5080b28b8efb1a2c9a82f6c","md5":"51f2fc33ea87ab2d163df3b6cb6da934","sha256":"deb03fa335f3b58ba4963bfb2f1337ee3b8571ac667eace3be51bd3572a401c4"},"downloads":-1,"filename":"exceptionman-0.0.0.dev11.tar.gz","has_sig":false,"md5_digest":"51f2fc33ea87ab2d163df3b6cb6da934","packagetype":"sdist","python_version":"source","requires_python":">=3.10","size":8141,"upload_time":"2024-09-24T10:54:35","upload_time_iso_8601":"2024-09-24T10:54:35.960684Z","url":"https://files.pythonhosted.org/packages/8e/03/93a77bd7662d46da63dcfbefeeb4470d9424d5080b28b8efb1a2c9a82f6c/exceptionman-0.0.0.dev11.tar.gz","yanked":false,"yanked_reason":null}],"vulnerabilities":[]},"0.0.0.dev12":{"info":{"author":null,"author_email":null,"bugtrack_url":null,"classifiers":[],"description_content_type":null,"docs_url":null,"download_url":null,"downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":null,"keywords":null,"license":null,"maintainer":null,"maintainer_email":null,"name":"ExceptionMan","package_url":"https://pypi.org/project/ExceptionMan/","platform":null,"project_url":"https://pypi.org/project/ExceptionMan/","project_urls":null,"provides_extra":null,"release_url":"https://pypi.org/project/ExceptionMan/0.0.0.dev12/","requires_dist":["MDit==0.0.0.dev12","rich>=13.5","pygments"],"requires_python":">=3.10","summary":null,"version":"0.0.0.dev12","yanked":false,"yanked_reason":null},"last_serial":25913582,"urls":[{"comment_text":"","digests":{"blake2b_256":"eb6909890fb9be0b533df45d28aac77a3c11ce0495ffdfaba42c566ab310ed28","md5":"23abc3c7a594ac2479f400723e2fa898","sha256":"42527529e1758012dc1ca96a9f52dfe4f89a60d7ddeb568b0c03bd956f8575cd"},"downloads":-1,"filename":"ExceptionMan-0.0.0.dev12-py3-none-any.whl","has_sig":false,"md5_digest":"23abc3c7a594ac2479f400723e2fa898","packagetype":"bdist_wheel","python_version":"py3","requires_python":">=3.10","size":9044,"upload_time":"2024-09-25T12:17:49","upload_time_iso_8601":"2024-09-25T12:17:49.740369Z","url":"https://files.pythonhosted.org/packages/eb/69/09890fb9be0b533df45d28aac77a3c11ce0495ffdfaba42c566ab310ed28/ExceptionMan-0.0.0.dev12-py3-none-any.whl","yanked":false,"yanked_reason":null},{"comment_text":"","digests":{"blake2b_256":"08fdc5c268f3d846e3350d78ef185c97cd8e7f357f2a698a5441331b4917f066","md5":"7d1adbacda6519629f7f50f44554150c","sha256":"c0f214fe5960eae98fafd183e42b86085c88606da0b1d106380a4cbe99900a75"},"downloads":-1,"filename":"exceptionman-0.0.0.dev12.tar.gz","has_sig":false,"md5_digest":"7d1adbacda6519629f7f50f44554150c","packagetype":"sdist","python_version":"source","requires_python":">=3.10","size":8143,"upload_time":"2024-09-25T12:17:50","upload_time_iso_8601":"2024-09-25T12:17:50.913728Z","url":"https://files.pythonhosted.org/packages/08/fd/c5c268f3d846e3350d78ef185c97cd8e7f357f2a698a5441331b4917f066/exceptionman-0.0.0.dev12.tar.gz","yanked":false,"yanked_reason":null}],"vulnerabilities":[]},"0.0.0.dev13":{"info":{"author":null,"author_email":null,"bugtrack_url":null,"classifiers":[],"description_content_type":null,"docs_url":null,"download_url":null,"downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":null,"keywords":null,"license":null,"maintainer":null,"maintainer_email":null,"name":"ExceptionMan","package_url":"https://pypi.org/project/ExceptionMan/","platform":null,"project_url":"https://pypi.org/project/ExceptionMan/","project_urls":null,"provides_extra":null,"release_url":"https://pypi.org/project/ExceptionMan/0.0.0.dev13/","requires_dist":["MDit==0.0.0.dev13","rich>=13.5","pygments"],"requires_python":">=3.10","summary":null,"version":"0.0.0.dev13","yanked":false,"yanked_reason":null},"last_serial":25913582,"urls":[{"comment_text":"","digests":{"blake2b_256":"948032c25e35af3079856a16f075fd1dd3b010f70ad36f73f5100a6cdaaaeb1f","md5":"108e96106c2c6091e3e87cfbbabb8f86","sha256":"b7db6c5cd3fc8bdfb78377185524c1ede337fc1a98648620f8dc991218b60523"},"downloads":-1,"filename":"ExceptionMan-0.0.0.dev13-py3-none-any.whl","has_sig":false,"md5_digest":"108e96106c2c6091e3e87cfbbabb8f86","packagetype":"bdist_wheel","python_version":"py3","requires_python":">=3.10","size":9042,"upload_time":"2024-09-26T00:21:05","upload_time_iso_8601":"2024-09-26T00:21:05.581717Z","url":"https://files.pythonhosted.org/packages/94/80/32c25e35af3079856a16f075fd1dd3b010f70ad36f73f5100a6cdaaaeb1f/ExceptionMan-0.0.0.dev13-py3-none-any.whl","yanked":false,"yanked_reason":null},{"comment_text":"","digests":{"blake2b_256":"3668792f36ee25e7dcb0309ec85ee940c326848f6bafc53524248a63f66d04ba","md5":"d5048b6da56a1f9f6a687a7998b3de18","sha256":"8fbcc7c6dc82eecb2188c1fb1abb3c9ff9a15b07f6f9b29a528a8d53321d3530"},"downloads":-1,"filename":"exceptionman-0.0.0.dev13.tar.gz","has_sig":false,"md5_digest":"d5048b6da56a1f9f6a687a7998b3de18","packagetype":"sdist","python_version":"source","requires_python":">=3.10","size":8140,"upload_time":"2024-09-26T00:21:08","upload_time_iso_8601":"2024-09-26T00:21:08.474224Z","url":"https://files.pythonhosted.org/packages/36/68/792f36ee25e7dcb0309ec85ee940c326848f6bafc53524248a63f66d04ba/exceptionman-0.0.0.dev13.tar.gz","yanked":false,"yanked_reason":null}],"vulnerabilities":[]},"0.0.0.dev14":{"info":{"author":null,"author_email":null,"bugtrack_url":null,"classifiers":[],"description_content_type":null,"docs_url":null,"download_url":null,"downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":null,"keywords":null,"license":null,"maintainer":null,"maintainer_email":null,"name":"ExceptionMan","package_url":"https://pypi.org/project/ExceptionMan/","platform":null,"project_url":"https://pypi.org/project/ExceptionMan/","project_urls":null,"provides_extra":null,"release_url":"https://pypi.org/project/ExceptionMan/0.0.0.dev14/","requires_dist":["MDit==0.0.0.dev14","rich>=13.5","pygments"],"requires_python":">=3.10","summary":null,"version":"0.0.0.dev14","yanked":false,"yanked_reason":null},"last_serial":25913582,"urls":[{"comment_text":"","digests":{"blake2b_256":"4fcab4f40e76390f9e209686938b91efdf781c5ac0516eebad3e5afddf7f9ce7","md5":"8dbcda27e6e01fe96d0fdb4d940baed4","sha256":"b71c1cd133b79a6fe618a38d8388bba844df69ea51949680f7e18d6bef1e6e5a"},"downloads":-1,"filename":"ExceptionMan-0.0.0.dev14-py3-none-any.whl","has_sig":false,"md5_digest":"8dbcda27e6e01fe96d0fdb4d940baed4","packagetype":"bdist_wheel","python_version":"py3","requires_python":">=3.10","size":8794,"upload_time":"2024-09-26T20:29:38","upload_time_iso_8601":"2024-09-26T20:29:38.978336Z","url":"https://files.pythonhosted.org/packages/4f/ca/b4f40e76390f9e209686938b91efdf781c5ac0516eebad3e5afddf7f9ce7/ExceptionMan-0.0.0.dev14-py3-none-any.whl","yanked":false,"yanked_reason":null},{"comment_text":"","digests":{"blake2b_256":"13aa49f393b4793a1e21632e72692ded7fac2a947f973a14b5fca68379905e3b","md5":"9bf6d999fb715731c1b27a3dc0733206","sha256":"ad079971fb10a94c85f9719d28d24fe640fcfb46569db6fdb816a6993ee3e99e"},"downloads":-1,"filename":"exceptionman-0.0.0.dev14.tar.gz","has_sig":false,"md5_digest":"9bf6d999fb715731c1b27a3dc0733206","packagetype":"sdist","python_version":"source","requires_python":">=3.10","size":7923,"upload_time":"2024-09-26T20:29:40","upload_time_iso_8601":"2024-09-26T20:29:40.431048Z","url":"https://files.pythonhosted.org/packages/13/aa/49f393b4793a1e21632e72692ded7fac2a947f973a14b5fca68379905e3b/exceptionman-0.0.0.dev14.tar.gz","yanked":false,"yanked_reason":null}],"vulnerabilities":[]},"0.0.0.dev15":{"info":{"author":null,"author_email":null,"bugtrack_url":null,"classifiers":[],"description_content_type":null,"docs_url":null,"download_url":null,"downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":null,"keywords":null,"license":null,"maintainer":null,"maintainer_email":null,"name":"ExceptionMan","package_url":"https://pypi.org/project/ExceptionMan/","platform":null,"project_url":"https://pypi.org/project/ExceptionMan/","project_urls":null,"provides_extra":null,"release_url":"https://pypi.org/project/ExceptionMan/0.0.0.dev15/","requires_dist":["MDit==0.0.0.dev15","rich>=13.5","pygments"],"requires_python":">=3.10","summary":null,"version":"0.0.0.dev15","yanked":false,"yanked_reason":null},"last_serial":25913582,"urls":[{"comment_text":"","digests":{"blake2b_256":"c7d0c341862d4ecff907cf932a2ab3017ed41535ce8913c1c69d53bf6a8d17d2","md5":"87bcd475c100168b97de3f3531acdbd1","sha256":"1e0c9ab2fecf7f76a9dc404f8b85cc2a2bfbff5c81c2d0ecc30032dc4a4cf0ae"},"downloads":-1,"filename":"ExceptionMan-0.0.0.dev15-py3-none-any.whl","has_sig":false,"md5_digest":"87bcd475c100168b97de3f3531acdbd1","packagetype":"bdist_wheel","python_version":"py3","requires_python":">=3.10","size":8792,"upload_time":"2024-09-27T01:04:28","upload_time_iso_8601":"2024-09-27T01:04:28.041867Z","url":"https://files.pythonhosted.org/packages/c7/d0/c341862d4ecff907cf932a2ab3017ed41535ce8913c1c69d53bf6a8d17d2/ExceptionMan-0.0.0.dev15-py3-none-any.whl","yanked":false,"yanked_reason":null},{"comment_text":"","digests":{"blake2b_256":"af48b8dd6e23361716dcd571c181f068d2de403f349492404870839a4ab969a4","md5":"e03241eb10ea398bec8802bede5b1e76","sha256":"138ff225365d1bbcf720779883b226b104dac653f69d128eba472d964569172a"},"downloads":-1,"filename":"exceptionman-0.0.0.dev15.tar.gz","has_sig":false,"md5_digest":"e03241eb10ea398bec8802bede5b1e76","packagetype":"sdist","python_version":"source","requires_python":">=3.10","size":7925,"upload_time":"2024-09-27T01:04:30","upload_time_iso_8601":"2024-09-27T01:04:30.109123Z","url":"https://files.pythonhosted.org/packages/af/48/b8dd6e23361716dcd571c181f068d2de403f349492404870839a4ab969a4/exceptionman-0.0.0.dev15.tar.gz","yanked":false,"yanked_reason":null}],"vulnerabilities":[]},"0.0.0.dev16":{"info":{"author":null,"author_email":null,"bugtrack_url":null,"classifiers":[],"description_content_type":null,"docs_url":null,"download_url":null,"downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":null,"keywords":null,"license":null,"maintainer":null,"maintainer_email":null,"name":"ExceptionMan","package_url":"https://pypi.org/project/ExceptionMan/","platform":null,"project_url":"https://pypi.org/project/ExceptionMan/","project_urls":null,"provides_extra":null,"release_url":"https://pypi.org/project/ExceptionMan/0.0.0.dev16/","requires_dist":["MDit==0.0.0.dev16","rich>=13.5","pygments"],"requires_python":">=3.10","summary":null,"version":"0.0.0.dev16","yanked":false,"yanked_reason":null},"last_serial":25913582,"urls":[{"comment_text":"","digests":{"blake2b_256":"f8072ac9aaa176cf9982a302d0f12f35ae996be938a03e1dd3e7843fcf279216","md5":"d3e622a1873e6692059899198118096b","sha256":"14ba6607bed895504488d7b2e4a26e2eee3c5dcd63d057bd85974f57665f82a6"},"downloads":-1,"filename":"ExceptionMan-0.0.0.dev16-py3-none-any.whl","has_sig":false,"md5_digest":"d3e622a1873e6692059899198118096b","packagetype":"bdist_wheel","python_version":"py3","requires_python":">=3.10","size":8795,"upload_time":"2024-09-29T20:05:27","upload_time_iso_8601":"2024-09-29T20:05:27.597528Z","url":"https://files.pythonhosted.org/packages/f8/07/2ac9aaa176cf9982a302d0f12f35ae996be938a03e1dd3e7843fcf279216/ExceptionMan-0.0.0.dev16-py3-none-any.whl","yanked":false,"yanked_reason":null},{"comment_text":"","digests":{"blake2b_256":"ebf4373052b30966f033d5b93febacef39886a02623bd37973f81af4847999f1","md5":"99ed0caea17019e41536d46d353f75e4","sha256":"da0b18a58e3e64bc7c29dbac7ba52fcdfda23ab67739dd2384a03b1a86c08181"},"downloads":-1,"filename":"exceptionman-0.0.0.dev16.tar.gz","has_sig":false,"md5_digest":"99ed0caea17019e41536d46d353f75e4","packagetype":"sdist","python_version":"source","requires_python":">=3.10","size":7920,"upload_time":"2024-09-29T20:05:28","upload_time_iso_8601":"2024-09-29T20:05:28.509104Z","url":"https://files.pythonhosted.org/packages/eb/f4/373052b30966f033d5b93febacef39886a02623bd37973f81af4847999f1/exceptionman-0.0.0.dev16.tar.gz","yanked":false,"yanked_reason":null}],"vulnerabilities":[]},"0.0.0.dev17":{"info":{"author":null,"author_email":null,"bugtrack_url":null,"classifiers":[],"description_content_type":null,"docs_url":null,"download_url":null,"downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":null,"keywords":null,"license":null,"maintainer":null,"maintainer_email":null,"name":"ExceptionMan","package_url":"https://pypi.org/project/ExceptionMan/","platform":null,"project_url":"https://pypi.org/project/ExceptionMan/","project_urls":null,"provides_extra":null,"release_url":"https://pypi.org/project/ExceptionMan/0.0.0.dev17/","requires_dist":["MDit==0.0.0.dev17","rich>=13.5","pygments"],"requires_python":">=3.10","summary":null,"version":"0.0.0.dev17","yanked":false,"yanked_reason":null},"last_serial":25913582,"urls":[{"comment_text":"","digests":{"blake2b_256":"808e1286e9e038f0b1b2774297674316b93c1415ae86b601d9554e3e2c7313a2","md5":"4b8f0f0692c852ea7f9a95f00f8a1807","sha256":"6d935a1c2211ab984156fe07ad94d958b3509eaae7e67a2b6fdf93779c248e75"},"downloads":-1,"filename":"ExceptionMan-0.0.0.dev17-py3-none-any.whl","has_sig":false,"md5_digest":"4b8f0f0692c852ea7f9a95f00f8a1807","packagetype":"bdist_wheel","python_version":"py3","requires_python":">=3.10","size":8794,"upload_time":"2024-09-30T18:49:44","upload_time_iso_8601":"2024-09-30T18:49:44.810092Z","url":"https://files.pythonhosted.org/packages/80/8e/1286e9e038f0b1b2774297674316b93c1415ae86b601d9554e3e2c7313a2/ExceptionMan-0.0.0.dev17-py3-none-any.whl","yanked":false,"yanked_reason":null},{"comment_text":"","digests":{"blake2b_256":"1879f3ae9cfcb66bc58c54c7edfb530cbe5098039c54bff910e375a8e4f27833","md5":"ff33b30a31244cf798e4d99e3c2b0dbf","sha256":"50250a0a1c666c07f67a80eb6823f4daa21e2e5b5c9099883c346eb9b126c9b3"},"downloads":-1,"filename":"exceptionman-0.0.0.dev17.tar.gz","has_sig":false,"md5_digest":"ff33b30a31244cf798e4d99e3c2b0dbf","packagetype":"sdist","python_version":"source","requires_python":">=3.10","size":7923,"upload_time":"2024-09-30T18:49:46","upload_time_iso_8601":"2024-09-30T18:49:46.942826Z","url":"https://files.pythonhosted.org/packages/18/79/f3ae9cfcb66bc58c54c7edfb530cbe5098039c54bff910e375a8e4f27833/exceptionman-0.0.0.dev17.tar.gz","yanked":false,"yanked_reason":null}],"vulnerabilities":[]},"0.0.0.dev18":{"info":{"author":null,"author_email":null,"bugtrack_url":null,"classifiers":[],"description_content_type":null,"docs_url":null,"download_url":null,"downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":null,"keywords":null,"license":null,"maintainer":null,"maintainer_email":null,"name":"ExceptionMan","package_url":"https://pypi.org/project/ExceptionMan/","platform":null,"project_url":"https://pypi.org/project/ExceptionMan/","project_urls":null,"provides_extra":null,"release_url":"https://pypi.org/project/ExceptionMan/0.0.0.dev18/","requires_dist":["MDit==0.0.0.dev18","rich>=13.5","pygments"],"requires_python":">=3.10","summary":null,"version":"0.0.0.dev18","yanked":false,"yanked_reason":null},"last_serial":25913582,"urls":[{"comment_text":"","digests":{"blake2b_256":"52447fba39349487b4bc0901d5093aae16fb422896ffc684a033b71ee3d6f998","md5":"e10a6dd80f8dae9ae6c87bf0a449d064","sha256":"48c14eceba6d5486605a27514c28f295f471f200e7f11caf447c1cf7e2e590be"},"downloads":-1,"filename":"ExceptionMan-0.0.0.dev18-py3-none-any.whl","has_sig":false,"md5_digest":"e10a6dd80f8dae9ae6c87bf0a449d064","packagetype":"bdist_wheel","python_version":"py3","requires_python":">=3.10","size":8792,"upload_time":"2024-09-30T19:04:47","upload_time_iso_8601":"2024-09-30T19:04:47.573978Z","url":"https://files.pythonhosted.org/packages/52/44/7fba39349487b4bc0901d5093aae16fb422896ffc684a033b71ee3d6f998/ExceptionMan-0.0.0.dev18-py3-none-any.whl","yanked":false,"yanked_reason":null},{"comment_text":"","digests":{"blake2b_256":"31b261d5d8c28c32bbc2731e0208d8f388bde4b2bf19b218e2c1d234291fa8c4","md5":"fa66fc3a5951634423ac01c3d170799d","sha256":"cf246cf9bcd73a7b858f4b1a3fb2e165b9c378f24f0953e954a84aa163df6441"},"downloads":-1,"filename":"exceptionman-0.0.0.dev18.tar.gz","has_sig":false,"md5_digest":"fa66fc3a5951634423ac01c3d170799d","packagetype":"sdist","python_version":"source","requires_python":">=3.10","size":7920,"upload_time":"2024-09-30T19:04:49","upload_time_iso_8601":"2024-09-30T19:04:49.119910Z","url":"https://files.pythonhosted.org/packages/31/b2/61d5d8c28c32bbc2731e0208d8f388bde4b2bf19b218e2c1d234291fa8c4/exceptionman-0.0.0.dev18.tar.gz","yanked":false,"yanked_reason":null}],"vulnerabilities":[]},"0.0.0.dev19":{"info":{"author":null,"author_email":null,"bugtrack_url":null,"classifiers":[],"description_content_type":null,"docs_url":null,"download_url":null,"downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":null,"keywords":null,"license":null,"maintainer":null,"maintainer_email":null,"name":"ExceptionMan","package_url":"https://pypi.org/project/ExceptionMan/","platform":null,"project_url":"https://pypi.org/project/ExceptionMan/","project_urls":null,"provides_extra":null,"release_url":"https://pypi.org/project/ExceptionMan/0.0.0.dev19/","requires_dist":["MDit==0.0.0.dev19","rich>=13.5","pygments"],"requires_python":">=3.10","summary":null,"version":"0.0.0.dev19","yanked":false,"yanked_reason":null},"last_serial":25913582,"urls":[{"comment_text":"","digests":{"blake2b_256":"cf125a982db2b064f2059cf926fd58a096a81582cd5cb2609c26fe1612218b47","md5":"756472f7a5e667674cb4afa68c0c8028","sha256":"56512a7f8d264c166998c61bee65f9225d5d4ae94eadf840e49c2e741ac6b827"},"downloads":-1,"filename":"ExceptionMan-0.0.0.dev19-py3-none-any.whl","has_sig":false,"md5_digest":"756472f7a5e667674cb4afa68c0c8028","packagetype":"bdist_wheel","python_version":"py3","requires_python":">=3.10","size":8793,"upload_time":"2024-10-01T19:14:47","upload_time_iso_8601":"2024-10-01T19:14:47.140245Z","url":"https://files.pythonhosted.org/packages/cf/12/5a982db2b064f2059cf926fd58a096a81582cd5cb2609c26fe1612218b47/ExceptionMan-0.0.0.dev19-py3-none-any.whl","yanked":false,"yanked_reason":null},{"comment_text":"","digests":{"blake2b_256":"c712f6afec8584252d932ca70490517ad63d2e9723cbd1b328988d5e45285ab5","md5":"5ad232efafec2398d72caa65bde34ca0","sha256":"ba7cfc2afb5888bffa3b82056831a403ba1d0182f3fb68f6f21363abcced7553"},"downloads":-1,"filename":"exceptionman-0.0.0.dev19.tar.gz","has_sig":false,"md5_digest":"5ad232efafec2398d72caa65bde34ca0","packagetype":"sdist","python_version":"source","requires_python":">=3.10","size":7936,"upload_time":"2024-10-01T19:14:50","upload_time_iso_8601":"2024-10-01T19:14:50.890241Z","url":"https://files.pythonhosted.org/packages/c7/12/f6afec8584252d932ca70490517ad63d2e9723cbd1b328988d5e45285ab5/exceptionman-0.0.0.dev19.tar.gz","yanked":false,"yanked_reason":null}],"vulnerabilities":[]},"0.0.0.dev2":{"info":{"author":null,"author_email":null,"bugtrack_url":null,"classifiers":[],"description_content_type":null,"docs_url":null,"download_url":null,"downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":null,"keywords":null,"license":null,"maintainer":null,"maintainer_email":null,"name":"ExceptionMan","package_url":"https://pypi.org/project/ExceptionMan/","platform":null,"project_url":"https://pypi.org/project/ExceptionMan/","project_urls":null,"provides_extra":null,"release_url":"https://pypi.org/project/ExceptionMan/0.0.0.dev2/","requires_dist":["mdit"],"requires_python":">=3.10","summary":null,"version":"0.0.0.dev2","yanked":false,"yanked_reason":null},"last_serial":25913582,"urls":[{"comment_text":"","digests":{"blake2b_256":"a02c0d52948a3fbad912936cc810cd95136f5d29a1e927a5c2d506a52728c364","md5":"32c90205318768f67fd2c40d4eab0d95","sha256":"b4ff9bfb2cbb95ca50d478874115be860b145e9f5eb5a60d3e8583628f1220d5"},"downloads":-1,"filename":"ExceptionMan-0.0.0.dev2-py3-none-any.whl","has_sig":false,"md5_digest":"32c90205318768f67fd2c40d4eab0d95","packagetype":"bdist_wheel","python_version":"py3","requires_python":">=3.10","size":1859,"upload_time":"2024-09-10T23:32:05","upload_time_iso_8601":"2024-09-10T23:32:05.478364Z","url":"https://files.pythonhosted.org/packages/a0/2c/0d52948a3fbad912936cc810cd95136f5d29a1e927a5c2d506a52728c364/ExceptionMan-0.0.0.dev2-py3-none-any.whl","yanked":false,"yanked_reason":null},{"comment_text":"","digests":{"blake2b_256":"82b6cc829997e658dca681107b60eb2ba6367455cb1b3cbe2d2086e57b6b09af","md5":"842b3dc4a909bae4342045e421dc66d7","sha256":"275af7015b9fe6ef9df3739911080f9414aaa778a147374b4b5e2acde8481084"},"downloads":-1,"filename":"exceptionman-0.0.0.dev2.tar.gz","has_sig":false,"md5_digest":"842b3dc4a909bae4342045e421dc66d7","packagetype":"sdist","python_version":"source","requires_python":">=3.10","size":1543,"upload_time":"2024-09-10T23:32:06","upload_time_iso_8601":"2024-09-10T23:32:06.854074Z","url":"https://files.pythonhosted.org/packages/82/b6/cc829997e658dca681107b60eb2ba6367455cb1b3cbe2d2086e57b6b09af/exceptionman-0.0.0.dev2.tar.gz","yanked":false,"yanked_reason":null}],"vulnerabilities":[]},"0.0.0.dev20":{"info":{"author":null,"author_email":null,"bugtrack_url":null,"classifiers":[],"description_content_type":null,"docs_url":null,"download_url":null,"downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":null,"keywords":null,"license":null,"maintainer":null,"maintainer_email":null,"name":"ExceptionMan","package_url":"https://pypi.org/project/ExceptionMan/","platform":null,"project_url":"https://pypi.org/project/ExceptionMan/","project_urls":null,"provides_extra":null,"release_url":"https://pypi.org/project/ExceptionMan/0.0.0.dev20/","requires_dist":["MDit==0.0.0.dev20","rich>=13.5","pygments"],"requires_python":">=3.10","summary":null,"version":"0.0.0.dev20","yanked":false,"yanked_reason":null},"last_serial":25913582,"urls":[{"comment_text":"","digests":{"blake2b_256":"4bd9dc7ada4a0c9233b219b1aa5f85767689a6b58875296864185fdd2214d26a","md5":"9da069275c7f1cf91ac2ab3d5311328d","sha256":"5cd4ca36df25c82f8964c5136a2cb690ba89456d5e0163e50888cdfaf38bbccf"},"downloads":-1,"filename":"ExceptionMan-0.0.0.dev20-py3-none-any.whl","has_sig":false,"md5_digest":"9da069275c7f1cf91ac2ab3d5311328d","packagetype":"bdist_wheel","python_version":"py3","requires_python":">=3.10","size":8794,"upload_time":"2024-10-03T18:02:53","upload_time_iso_8601":"2024-10-03T18:02:53.510553Z","url":"https://files.pythonhosted.org/packages/4b/d9/dc7ada4a0c9233b219b1aa5f85767689a6b58875296864185fdd2214d26a/ExceptionMan-0.0.0.dev20-py3-none-any.whl","yanked":false,"yanked_reason":null},{"comment_text":"","digests":{"blake2b_256":"dca72d8ad5ad3c904017deb0e1d7e2fb277122204fffe15d42b3cdf1b050ebac","md5":"99de97a10d36f92a0f058100b8d2eb89","sha256":"7522986a6520a63f9573f06567830af4e0b66b334987e8a280cfea3b024fcca3"},"downloads":-1,"filename":"exceptionman-0.0.0.dev20.tar.gz","has_sig":false,"md5_digest":"99de97a10d36f92a0f058100b8d2eb89","packagetype":"sdist","python_version":"source","requires_python":">=3.10","size":7937,"upload_time":"2024-10-03T18:02:55","upload_time_iso_8601":"2024-10-03T18:02:55.481922Z","url":"https://files.pythonhosted.org/packages/dc/a7/2d8ad5ad3c904017deb0e1d7e2fb277122204fffe15d42b3cdf1b050ebac/exceptionman-0.0.0.dev20.tar.gz","yanked":false,"yanked_reason":null}],"vulnerabilities":[]},"0.0.0.dev21":{"info":{"author":null,"author_email":null,"bugtrack_url":null,"classifiers":[],"description_content_type":null,"docs_url":null,"download_url":null,"downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":null,"keywords":null,"license":null,"maintainer":null,"maintainer_email":null,"name":"ExceptionMan","package_url":"https://pypi.org/project/ExceptionMan/","platform":null,"project_url":"https://pypi.org/project/ExceptionMan/","project_urls":null,"provides_extra":null,"release_url":"https://pypi.org/project/ExceptionMan/0.0.0.dev21/","requires_dist":["MDit==0.0.0.dev21","rich>=13.5","pygments"],"requires_python":">=3.10","summary":null,"version":"0.0.0.dev21","yanked":false,"yanked_reason":null},"last_serial":25913582,"urls":[{"comment_text":"","digests":{"blake2b_256":"f712cb83a8838370c240725a81bcf917f5c0cd0d1ee313e030ba2ebcd742cb3b","md5":"1127df15700a134474694790c64ff2a4","sha256":"36cc2f2b1cedf69da8e68d548216cfd591172c3511dc47af3c581d1d18df4d59"},"downloads":-1,"filename":"ExceptionMan-0.0.0.dev21-py3-none-any.whl","has_sig":false,"md5_digest":"1127df15700a134474694790c64ff2a4","packagetype":"bdist_wheel","python_version":"py3","requires_python":">=3.10","size":8793,"upload_time":"2024-10-13T08:16:04","upload_time_iso_8601":"2024-10-13T08:16:04.876687Z","url":"https://files.pythonhosted.org/packages/f7/12/cb83a8838370c240725a81bcf917f5c0cd0d1ee313e030ba2ebcd742cb3b/ExceptionMan-0.0.0.dev21-py3-none-any.whl","yanked":false,"yanked_reason":null},{"comment_text":"","digests":{"blake2b_256":"cf6b75a3755ac860f04cb914f1cb6b825851bf4b5c2396a7cfb5d77b9ec28025","md5":"616d7bef9b02d37ba1169b8dc7f6d364","sha256":"4d4dac6c9e8cc50aa4e986e7aa2358bd102c44612c0ff8cf83814bddaea4bc99"},"downloads":-1,"filename":"exceptionman-0.0.0.dev21.tar.gz","has_sig":false,"md5_digest":"616d7bef9b02d37ba1169b8dc7f6d364","packagetype":"sdist","python_version":"source","requires_python":">=3.10","size":7915,"upload_time":"2024-10-13T08:16:06","upload_time_iso_8601":"2024-10-13T08:16:06.092593Z","url":"https://files.pythonhosted.org/packages/cf/6b/75a3755ac860f04cb914f1cb6b825851bf4b5c2396a7cfb5d77b9ec28025/exceptionman-0.0.0.dev21.tar.gz","yanked":false,"yanked_reason":null}],"vulnerabilities":[]},"0.0.0.dev22":{"info":{"author":null,"author_email":null,"bugtrack_url":null,"classifiers":[],"description_content_type":null,"docs_url":null,"download_url":null,"downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":null,"keywords":null,"license":null,"maintainer":null,"maintainer_email":null,"name":"ExceptionMan","package_url":"https://pypi.org/project/ExceptionMan/","platform":null,"project_url":"https://pypi.org/project/ExceptionMan/","project_urls":null,"provides_extra":null,"release_url":"https://pypi.org/project/ExceptionMan/0.0.0.dev22/","requires_dist":["MDit==0.0.0.dev22","rich>=13.5","pygments"],"requires_python":">=3.10","summary":null,"version":"0.0.0.dev22","yanked":false,"yanked_reason":null},"last_serial":25913582,"urls":[{"comment_text":"","digests":{"blake2b_256":"2150ccac88cc86bf282767ebf242431d1f6783dd4b59f1317a5c2be6215d31f4","md5":"528ac3bc73c14017633c14893759e854","sha256":"9661468fc2395bb20da2854461d05b1d99c69532c9dfa401aa6b4fa3fdffc357"},"downloads":-1,"filename":"ExceptionMan-0.0.0.dev22-py3-none-any.whl","has_sig":false,"md5_digest":"528ac3bc73c14017633c14893759e854","packagetype":"bdist_wheel","python_version":"py3","requires_python":">=3.10","size":8793,"upload_time":"2024-10-13T20:53:29","upload_time_iso_8601":"2024-10-13T20:53:29.600849Z","url":"https://files.pythonhosted.org/packages/21/50/ccac88cc86bf282767ebf242431d1f6783dd4b59f1317a5c2be6215d31f4/ExceptionMan-0.0.0.dev22-py3-none-any.whl","yanked":false,"yanked_reason":null},{"comment_text":"","digests":{"blake2b_256":"c786009fc8e55e484d1052c40ebb464fd1ebc8d542dda9e638a8e6cd4095ef76","md5":"77877d34dfa90034d8222b6e278f4f7b","sha256":"bc961b9f35e4f02ebda135858c0137975a9795ab99c9a232a3f603309c9c371e"},"downloads":-1,"filename":"exceptionman-0.0.0.dev22.tar.gz","has_sig":false,"md5_digest":"77877d34dfa90034d8222b6e278f4f7b","packagetype":"sdist","python_version":"source","requires_python":">=3.10","size":7910,"upload_time":"2024-10-13T20:53:31","upload_time_iso_8601":"2024-10-13T20:53:31.127727Z","url":"https://files.pythonhosted.org/packages/c7/86/009fc8e55e484d1052c40ebb464fd1ebc8d542dda9e638a8e6cd4095ef76/exceptionman-0.0.0.dev22.tar.gz","yanked":false,"yanked_reason":null}],"vulnerabilities":[]},"0.0.0.dev23":{"info":{"author":null,"author_email":null,"bugtrack_url":null,"classifiers":[],"description_content_type":null,"docs_url":null,"download_url":null,"downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":null,"keywords":null,"license":null,"maintainer":null,"maintainer_email":null,"name":"ExceptionMan","package_url":"https://pypi.org/project/ExceptionMan/","platform":null,"project_url":"https://pypi.org/project/ExceptionMan/","project_urls":null,"provides_extra":null,"release_url":"https://pypi.org/project/ExceptionMan/0.0.0.dev23/","requires_dist":["MDit==0.0.0.dev23","rich>=13.5","pygments"],"requires_python":">=3.10","summary":null,"version":"0.0.0.dev23","yanked":false,"yanked_reason":null},"last_serial":25913582,"urls":[{"comment_text":"","digests":{"blake2b_256":"51a06206b3782b94ff9e645882f1b88a8da00d2b901ab3c58a613ccf5606a582","md5":"8cf3513d807c5797a9d3d7aff4f0deae","sha256":"24d612b90198c41055f9054e66dcdc59b4debba0bcf832730d1fdbb3bccd83fc"},"downloads":-1,"filename":"ExceptionMan-0.0.0.dev23-py3-none-any.whl","has_sig":false,"md5_digest":"8cf3513d807c5797a9d3d7aff4f0deae","packagetype":"bdist_wheel","python_version":"py3","requires_python":">=3.10","size":8793,"upload_time":"2024-10-17T09:30:40","upload_time_iso_8601":"2024-10-17T09:30:40.489148Z","url":"https://files.pythonhosted.org/packages/51/a0/6206b3782b94ff9e645882f1b88a8da00d2b901ab3c58a613ccf5606a582/ExceptionMan-0.0.0.dev23-py3-none-any.whl","yanked":false,"yanked_reason":null},{"comment_text":"","digests":{"blake2b_256":"50e9a088cf4b9a6154300f224066d36757c9a20c43d9b159a7afbffa916765ac","md5":"3c012001b9db5dac01020bb42c7eda29","sha256":"f435cbde4aa542e526162e7c637e939ef789c7ea0bc3b93de9f54266ba732c70"},"downloads":-1,"filename":"exceptionman-0.0.0.dev23.tar.gz","has_sig":false,"md5_digest":"3c012001b9db5dac01020bb42c7eda29","packagetype":"sdist","python_version":"source","requires_python":">=3.10","size":7905,"upload_time":"2024-10-17T09:30:41","upload_time_iso_8601":"2024-10-17T09:30:41.307597Z","url":"https://files.pythonhosted.org/packages/50/e9/a088cf4b9a6154300f224066d36757c9a20c43d9b159a7afbffa916765ac/exceptionman-0.0.0.dev23.tar.gz","yanked":false,"yanked_reason":null}],"vulnerabilities":[]},"0.0.0.dev24":{"info":{"author":null,"author_email":null,"bugtrack_url":null,"classifiers":[],"description_content_type":null,"docs_url":null,"download_url":null,"downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":null,"keywords":null,"license":null,"maintainer":null,"maintainer_email":null,"name":"ExceptionMan","package_url":"https://pypi.org/project/ExceptionMan/","platform":null,"project_url":"https://pypi.org/project/ExceptionMan/","project_urls":null,"provides_extra":null,"release_url":"https://pypi.org/project/ExceptionMan/0.0.0.dev24/","requires_dist":["MDit==0.0.0.dev24","rich>=13.5","pygments"],"requires_python":">=3.10","summary":null,"version":"0.0.0.dev24","yanked":false,"yanked_reason":null},"last_serial":25913582,"urls":[{"comment_text":"","digests":{"blake2b_256":"5c2d20af6c9be1f75b96a125c2788665b11d5ac043d514ff56404d6080fee42c","md5":"245939bd99291ef790f3af7f4d043a0d","sha256":"48be50ed5f82e72acdbeeea2b1cd895466e0c3fcacc7ff882e30bea598dee7a8"},"downloads":-1,"filename":"ExceptionMan-0.0.0.dev24-py3-none-any.whl","has_sig":false,"md5_digest":"245939bd99291ef790f3af7f4d043a0d","packagetype":"bdist_wheel","python_version":"py3","requires_python":">=3.10","size":8794,"upload_time":"2024-10-17T10:32:56","upload_time_iso_8601":"2024-10-17T10:32:56.469300Z","url":"https://files.pythonhosted.org/packages/5c/2d/20af6c9be1f75b96a125c2788665b11d5ac043d514ff56404d6080fee42c/ExceptionMan-0.0.0.dev24-py3-none-any.whl","yanked":false,"yanked_reason":null},{"comment_text":"","digests":{"blake2b_256":"977883ed4fca8837b8cfaa7ee207ad01bee62f61761696af40708ed403258368","md5":"10e7c2930f1893e06ee48b1d21cbc5a3","sha256":"f4c2cae7000cf48950b8f5c987805f32afb00967658c2532cf9572999c77d494"},"downloads":-1,"filename":"exceptionman-0.0.0.dev24.tar.gz","has_sig":false,"md5_digest":"10e7c2930f1893e06ee48b1d21cbc5a3","packagetype":"sdist","python_version":"source","requires_python":">=3.10","size":7906,"upload_time":"2024-10-17T10:33:00","upload_time_iso_8601":"2024-10-17T10:33:00.113750Z","url":"https://files.pythonhosted.org/packages/97/78/83ed4fca8837b8cfaa7ee207ad01bee62f61761696af40708ed403258368/exceptionman-0.0.0.dev24.tar.gz","yanked":false,"yanked_reason":null}],"vulnerabilities":[]},"0.0.0.dev25":{"info":{"author":null,"author_email":null,"bugtrack_url":null,"classifiers":[],"description_content_type":null,"docs_url":null,"download_url":null,"downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":null,"keywords":null,"license":null,"maintainer":null,"maintainer_email":null,"name":"ExceptionMan","package_url":"https://pypi.org/project/ExceptionMan/","platform":null,"project_url":"https://pypi.org/project/ExceptionMan/","project_urls":null,"provides_extra":null,"release_url":"https://pypi.org/project/ExceptionMan/0.0.0.dev25/","requires_dist":["MDit==0.0.0.dev25","rich>=13.5","pygments"],"requires_python":">=3.10","summary":null,"version":"0.0.0.dev25","yanked":false,"yanked_reason":null},"last_serial":25913582,"urls":[{"comment_text":"","digests":{"blake2b_256":"72add01eb67e76e53e06f1da6f90cf07b5d7ec04dd0775712b467732420ccd90","md5":"40a4523142b9823a0190fa0842e1fb40","sha256":"c588c711151f845e8c118c89929b8ea6b7f1f062839f545d1a4c6e8cfdcd10bb"},"downloads":-1,"filename":"ExceptionMan-0.0.0.dev25-py3-none-any.whl","has_sig":false,"md5_digest":"40a4523142b9823a0190fa0842e1fb40","packagetype":"bdist_wheel","python_version":"py3","requires_python":">=3.10","size":8793,"upload_time":"2024-10-17T10:55:27","upload_time_iso_8601":"2024-10-17T10:55:27.637800Z","url":"https://files.pythonhosted.org/packages/72/ad/d01eb67e76e53e06f1da6f90cf07b5d7ec04dd0775712b467732420ccd90/ExceptionMan-0.0.0.dev25-py3-none-any.whl","yanked":false,"yanked_reason":null},{"comment_text":"","digests":{"blake2b_256":"dc95a6a8f87c392eec7e996dfa188a4d7c1ca5cd624191c0c3ed1361023af51f","md5":"4397638f762d69009d5656958fba34e0","sha256":"e961695ad75af887c983e9335eb327eecbcc1593e2c74bfdb246351b4064c117"},"downloads":-1,"filename":"exceptionman-0.0.0.dev25.tar.gz","has_sig":false,"md5_digest":"4397638f762d69009d5656958fba34e0","packagetype":"sdist","python_version":"source","requires_python":">=3.10","size":7909,"upload_time":"2024-10-17T10:55:28","upload_time_iso_8601":"2024-10-17T10:55:28.441353Z","url":"https://files.pythonhosted.org/packages/dc/95/a6a8f87c392eec7e996dfa188a4d7c1ca5cd624191c0c3ed1361023af51f/exceptionman-0.0.0.dev25.tar.gz","yanked":false,"yanked_reason":null}],"vulnerabilities":[]},"0.0.0.dev26":{"info":{"author":null,"author_email":null,"bugtrack_url":null,"classifiers":[],"description_content_type":null,"docs_url":null,"download_url":null,"downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":null,"keywords":null,"license":null,"maintainer":null,"maintainer_email":null,"name":"ExceptionMan","package_url":"https://pypi.org/project/ExceptionMan/","platform":null,"project_url":"https://pypi.org/project/ExceptionMan/","project_urls":null,"provides_extra":null,"release_url":"https://pypi.org/project/ExceptionMan/0.0.0.dev26/","requires_dist":["MDit==0.0.0.dev26","rich>=13.5","pygments"],"requires_python":">=3.10","summary":null,"version":"0.0.0.dev26","yanked":false,"yanked_reason":null},"last_serial":25913582,"urls":[{"comment_text":"","digests":{"blake2b_256":"5026f43123d67807e82732811d95d2a3130eab6147685ad0d04a333c19f65dbc","md5":"77e7bbac2eb4cf7a6a00eba674d64873","sha256":"6e4ac5e9cbe09d901d9e90209f330d4dbec7fb2e770400792e7967d6e716f2d5"},"downloads":-1,"filename":"ExceptionMan-0.0.0.dev26-py3-none-any.whl","has_sig":false,"md5_digest":"77e7bbac2eb4cf7a6a00eba674d64873","packagetype":"bdist_wheel","python_version":"py3","requires_python":">=3.10","size":8794,"upload_time":"2024-10-20T13:36:17","upload_time_iso_8601":"2024-10-20T13:36:17.398337Z","url":"https://files.pythonhosted.org/packages/50/26/f43123d67807e82732811d95d2a3130eab6147685ad0d04a333c19f65dbc/ExceptionMan-0.0.0.dev26-py3-none-any.whl","yanked":false,"yanked_reason":null},{"comment_text":"","digests":{"blake2b_256":"8e92d31f37b737e83b00cf3b6238823145d25730f64be58fc43c942ce03c8ba6","md5":"bac69c2bd96d749255a03d3190acaa10","sha256":"dc33ae5cb2264dfb4d1933932265d833b6e51ba3c21f5b93b892ee2fa5131b32"},"downloads":-1,"filename":"exceptionman-0.0.0.dev26.tar.gz","has_sig":false,"md5_digest":"bac69c2bd96d749255a03d3190acaa10","packagetype":"sdist","python_version":"source","requires_python":">=3.10","size":7915,"upload_time":"2024-10-20T13:36:18","upload_time_iso_8601":"2024-10-20T13:36:18.822597Z","url":"https://files.pythonhosted.org/packages/8e/92/d31f37b737e83b00cf3b6238823145d25730f64be58fc43c942ce03c8ba6/exceptionman-0.0.0.dev26.tar.gz","yanked":false,"yanked_reason":null}],"vulnerabilities":[]},"0.0.0.dev27":{"info":{"author":null,"author_email":null,"bugtrack_url":null,"classifiers":[],"description_content_type":null,"docs_url":null,"download_url":null,"downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":null,"keywords":null,"license":null,"maintainer":null,"maintainer_email":null,"name":"ExceptionMan","package_url":"https://pypi.org/project/ExceptionMan/","platform":null,"project_url":"https://pypi.org/project/ExceptionMan/","project_urls":null,"provides_extra":null,"release_url":"https://pypi.org/project/ExceptionMan/0.0.0.dev27/","requires_dist":["MDit==0.0.0.dev27","rich>=13.5","pygments"],"requires_python":">=3.10","summary":null,"version":"0.0.0.dev27","yanked":false,"yanked_reason":null},"last_serial":25913582,"urls":[{"comment_text":"","digests":{"blake2b_256":"8b40aad5aed0f2f77e71f6956853c6e530856de513174b2da983c24b1ae46fee","md5":"dae6746b7ba04e9f8d22f7212fc97007","sha256":"873eeeaef91ab9990cadfb8198c8a90ce5758d3b3d06228853951bd1e7c76b77"},"downloads":-1,"filename":"ExceptionMan-0.0.0.dev27-py3-none-any.whl","has_sig":false,"md5_digest":"dae6746b7ba04e9f8d22f7212fc97007","packagetype":"bdist_wheel","python_version":"py3","requires_python":">=3.10","size":8794,"upload_time":"2024-10-20T20:38:08","upload_time_iso_8601":"2024-10-20T20:38:08.258171Z","url":"https://files.pythonhosted.org/packages/8b/40/aad5aed0f2f77e71f6956853c6e530856de513174b2da983c24b1ae46fee/ExceptionMan-0.0.0.dev27-py3-none-any.whl","yanked":false,"yanked_reason":null},{"comment_text":"","digests":{"blake2b_256":"b74af99a19f07430a090e2489760a382649d9971447cd72722e39fd70ee67a0e","md5":"96d05ddb53c1b95ef112cb159301a67a","sha256":"2dca46006d674dcd4a5707734c54f5dbbd506c724bfa0b1b0f6820de71f5aae7"},"downloads":-1,"filename":"exceptionman-0.0.0.dev27.tar.gz","has_sig":false,"md5_digest":"96d05ddb53c1b95ef112cb159301a67a","packagetype":"sdist","python_version":"source","requires_python":">=3.10","size":7908,"upload_time":"2024-10-20T20:38:09","upload_time_iso_8601":"2024-10-20T20:38:09.414502Z","url":"https://files.pythonhosted.org/packages/b7/4a/f99a19f07430a090e2489760a382649d9971447cd72722e39fd70ee67a0e/exceptionman-0.0.0.dev27.tar.gz","yanked":false,"yanked_reason":null}],"vulnerabilities":[]},"0.0.0.dev28":{"info":{"author":null,"author_email":null,"bugtrack_url":null,"classifiers":[],"description_content_type":null,"docs_url":null,"download_url":null,"downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":null,"keywords":null,"license":null,"maintainer":null,"maintainer_email":null,"name":"ExceptionMan","package_url":"https://pypi.org/project/ExceptionMan/","platform":null,"project_url":"https://pypi.org/project/ExceptionMan/","project_urls":null,"provides_extra":null,"release_url":"https://pypi.org/project/ExceptionMan/0.0.0.dev28/","requires_dist":["MDit==0.0.0.dev28","rich>=13.5","pygments"],"requires_python":">=3.10","summary":null,"version":"0.0.0.dev28","yanked":false,"yanked_reason":null},"last_serial":25913582,"urls":[{"comment_text":"","digests":{"blake2b_256":"0aa32a137e935abe41fc7c21ad6ec5a7e7eac50c982a0179123d7cf7c3c974b9","md5":"3de33fbb846d8e5bb71e9290735e2e8e","sha256":"5b8a8e6bb364c4a576347998c78c9c6a32c9b95322dec3b8f4f6b7854da9da7d"},"downloads":-1,"filename":"ExceptionMan-0.0.0.dev28-py3-none-any.whl","has_sig":false,"md5_digest":"3de33fbb846d8e5bb71e9290735e2e8e","packagetype":"bdist_wheel","python_version":"py3","requires_python":">=3.10","size":8794,"upload_time":"2024-10-25T16:11:10","upload_time_iso_8601":"2024-10-25T16:11:10.900971Z","url":"https://files.pythonhosted.org/packages/0a/a3/2a137e935abe41fc7c21ad6ec5a7e7eac50c982a0179123d7cf7c3c974b9/ExceptionMan-0.0.0.dev28-py3-none-any.whl","yanked":false,"yanked_reason":null},{"comment_text":"","digests":{"blake2b_256":"d4f65769dbb26f9b3f239f2e69eb9a8872b238c05f59445523e334eeb9341279","md5":"a164dfdf4e42c8ae17c2c127dddb833d","sha256":"f70652bc19b4ac33c7458a91e63f3f56cf3e1bf3d15d8869edc4adceaebea5d6"},"downloads":-1,"filename":"exceptionman-0.0.0.dev28.tar.gz","has_sig":false,"md5_digest":"a164dfdf4e42c8ae17c2c127dddb833d","packagetype":"sdist","python_version":"source","requires_python":">=3.10","size":7916,"upload_time":"2024-10-25T16:11:13","upload_time_iso_8601":"2024-10-25T16:11:13.682563Z","url":"https://files.pythonhosted.org/packages/d4/f6/5769dbb26f9b3f239f2e69eb9a8872b238c05f59445523e334eeb9341279/exceptionman-0.0.0.dev28.tar.gz","yanked":false,"yanked_reason":null}],"vulnerabilities":[]},"0.0.0.dev29":{"info":{"author":null,"author_email":null,"bugtrack_url":null,"classifiers":[],"description_content_type":null,"docs_url":null,"download_url":null,"downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":null,"keywords":null,"license":null,"maintainer":null,"maintainer_email":null,"name":"ExceptionMan","package_url":"https://pypi.org/project/ExceptionMan/","platform":null,"project_url":"https://pypi.org/project/ExceptionMan/","project_urls":null,"provides_extra":null,"release_url":"https://pypi.org/project/ExceptionMan/0.0.0.dev29/","requires_dist":["MDit==0.0.0.dev29","rich>=13.5","pygments"],"requires_python":">=3.10","summary":null,"version":"0.0.0.dev29","yanked":false,"yanked_reason":null},"last_serial":25913582,"urls":[{"comment_text":"","digests":{"blake2b_256":"627bbbba4a3c7ebf6672615e15ee8feddef31903dfe9eed6d44c1a0f249b21b5","md5":"2ad2eb171fba3d84c5c8600c5612ec9a","sha256":"05860c3e64811b2416ea86687f10e85c1a6361d3b6bf06f2651c8759c3050b15"},"downloads":-1,"filename":"ExceptionMan-0.0.0.dev29-py3-none-any.whl","has_sig":false,"md5_digest":"2ad2eb171fba3d84c5c8600c5612ec9a","packagetype":"bdist_wheel","python_version":"py3","requires_python":">=3.10","size":8796,"upload_time":"2024-10-31T14:54:43","upload_time_iso_8601":"2024-10-31T14:54:43.535019Z","url":"https://files.pythonhosted.org/packages/62/7b/bbba4a3c7ebf6672615e15ee8feddef31903dfe9eed6d44c1a0f249b21b5/ExceptionMan-0.0.0.dev29-py3-none-any.whl","yanked":false,"yanked_reason":null},{"comment_text":"","digests":{"blake2b_256":"cee7032b3883a99fa3530be44fcc8605828310b375076807b990462be63021b0","md5":"96372960ab6fead202dfde764a7427bb","sha256":"132d70ca9ddd51cab157578c0fbed5d0878f4f40da85890e4c3d24c15290a5dc"},"downloads":-1,"filename":"exceptionman-0.0.0.dev29.tar.gz","has_sig":false,"md5_digest":"96372960ab6fead202dfde764a7427bb","packagetype":"sdist","python_version":"source","requires_python":">=3.10","size":7914,"upload_time":"2024-10-31T14:54:49","upload_time_iso_8601":"2024-10-31T14:54:49.585216Z","url":"https://files.pythonhosted.org/packages/ce/e7/032b3883a99fa3530be44fcc8605828310b375076807b990462be63021b0/exceptionman-0.0.0.dev29.tar.gz","yanked":false,"yanked_reason":null}],"vulnerabilities":[]},"0.0.0.dev3":{"info":{"author":null,"author_email":null,"bugtrack_url":null,"classifiers":[],"description_content_type":null,"docs_url":null,"download_url":null,"downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":null,"keywords":null,"license":null,"maintainer":null,"maintainer_email":null,"name":"ExceptionMan","package_url":"https://pypi.org/project/ExceptionMan/","platform":null,"project_url":"https://pypi.org/project/ExceptionMan/","project_urls":null,"provides_extra":null,"release_url":"https://pypi.org/project/ExceptionMan/0.0.0.dev3/","requires_dist":["mdit","rich","pygments"],"requires_python":">=3.10","summary":null,"version":"0.0.0.dev3","yanked":false,"yanked_reason":null},"last_serial":25913582,"urls":[{"comment_text":"","digests":{"blake2b_256":"2438b967b2bae9d6d80f805cb2b5fc57c248e987e6e6c1ced33e0a58268765e7","md5":"c4df8a2a21bfdf81af322d7a89e90ca8","sha256":"e2ccccae416744eac7e6b3141031bf4ddfeac72ca4fdc082b9c56df5e1876865"},"downloads":-1,"filename":"ExceptionMan-0.0.0.dev3-py3-none-any.whl","has_sig":false,"md5_digest":"c4df8a2a21bfdf81af322d7a89e90ca8","packagetype":"bdist_wheel","python_version":"py3","requires_python":">=3.10","size":9017,"upload_time":"2024-09-15T18:18:55","upload_time_iso_8601":"2024-09-15T18:18:55.582201Z","url":"https://files.pythonhosted.org/packages/24/38/b967b2bae9d6d80f805cb2b5fc57c248e987e6e6c1ced33e0a58268765e7/ExceptionMan-0.0.0.dev3-py3-none-any.whl","yanked":false,"yanked_reason":null},{"comment_text":"","digests":{"blake2b_256":"29706d39f036357e567c4e1caa1c0127b192415cbf1fe0a30470b413a2f28229","md5":"377a1f5a2d56297a8f9ac342cf513dbc","sha256":"d25b59b2d878f034d7a04dd470145548144dc0b85377c3b9db6137f3f6ba8165"},"downloads":-1,"filename":"exceptionman-0.0.0.dev3.tar.gz","has_sig":false,"md5_digest":"377a1f5a2d56297a8f9ac342cf513dbc","packagetype":"sdist","python_version":"source","requires_python":">=3.10","size":8112,"upload_time":"2024-09-15T18:18:56","upload_time_iso_8601":"2024-09-15T18:18:56.966145Z","url":"https://files.pythonhosted.org/packages/29/70/6d39f036357e567c4e1caa1c0127b192415cbf1fe0a30470b413a2f28229/exceptionman-0.0.0.dev3.tar.gz","yanked":false,"yanked_reason":null}],"vulnerabilities":[]},"0.0.0.dev30":{"info":{"author":null,"author_email":null,"bugtrack_url":null,"classifiers":[],"description_content_type":null,"docs_url":null,"download_url":null,"downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":null,"keywords":null,"license":null,"maintainer":null,"maintainer_email":null,"name":"ExceptionMan","package_url":"https://pypi.org/project/ExceptionMan/","platform":null,"project_url":"https://pypi.org/project/ExceptionMan/","project_urls":null,"provides_extra":null,"release_url":"https://pypi.org/project/ExceptionMan/0.0.0.dev30/","requires_dist":["MDit==0.0.0.dev30","rich>=13.5","pygments"],"requires_python":">=3.10","summary":null,"version":"0.0.0.dev30","yanked":false,"yanked_reason":null},"last_serial":25913582,"urls":[{"comment_text":"","digests":{"blake2b_256":"2d1a06b4d7b28d14f34427644989043dbe317ba9ab13b90b086c1715dc55a620","md5":"2329c083c6a74e786170943abc57b55b","sha256":"7b8d97363a18ba00150dc48d9f5ea502bbab27fc58c92d12247f2170be8f054c"},"downloads":-1,"filename":"ExceptionMan-0.0.0.dev30-py3-none-any.whl","has_sig":false,"md5_digest":"2329c083c6a74e786170943abc57b55b","packagetype":"bdist_wheel","python_version":"py3","requires_python":">=3.10","size":8794,"upload_time":"2024-11-01T20:09:29","upload_time_iso_8601":"2024-11-01T20:09:29.530430Z","url":"https://files.pythonhosted.org/packages/2d/1a/06b4d7b28d14f34427644989043dbe317ba9ab13b90b086c1715dc55a620/ExceptionMan-0.0.0.dev30-py3-none-any.whl","yanked":false,"yanked_reason":null},{"comment_text":"","digests":{"blake2b_256":"7bfe51bb8be8881787e1a7d927db28461d79d9117574f03c81cfb28b1e5bb33b","md5":"f1e1c7750c6f868e62b8ee2f228cb512","sha256":"8b85d609fd3d3bafd0e5286fd860d1468d3d656130d7c69d1d97c1d6a2f0ee97"},"downloads":-1,"filename":"exceptionman-0.0.0.dev30.tar.gz","has_sig":false,"md5_digest":"f1e1c7750c6f868e62b8ee2f228cb512","packagetype":"sdist","python_version":"source","requires_python":">=3.10","size":7928,"upload_time":"2024-11-01T20:09:32","upload_time_iso_8601":"2024-11-01T20:09:32.587375Z","url":"https://files.pythonhosted.org/packages/7b/fe/51bb8be8881787e1a7d927db28461d79d9117574f03c81cfb28b1e5bb33b/exceptionman-0.0.0.dev30.tar.gz","yanked":false,"yanked_reason":null}],"vulnerabilities":[]},"0.0.0.dev31":{"info":{"author":null,"author_email":null,"bugtrack_url":null,"classifiers":[],"description_content_type":null,"docs_url":null,"download_url":null,"downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":null,"keywords":null,"license":null,"maintainer":null,"maintainer_email":null,"name":"ExceptionMan","package_url":"https://pypi.org/project/ExceptionMan/","platform":null,"project_url":"https://pypi.org/project/ExceptionMan/","project_urls":null,"provides_extra":null,"release_url":"https://pypi.org/project/ExceptionMan/0.0.0.dev31/","requires_dist":["MDit==0.0.0.dev31","rich>=13.5","pygments"],"requires_python":">=3.10","summary":null,"version":"0.0.0.dev31","yanked":false,"yanked_reason":null},"last_serial":25913582,"urls":[{"comment_text":"","digests":{"blake2b_256":"e875907dc618f49999608e1c45df140b9c236f7696157fc79eb0b0fb73921e33","md5":"ca3792cf3510eb9cea84927847f85394","sha256":"73dc1ef0c65d273526c6c3a6af84e8bef44b4d026bde99452c64cef5b4afb147"},"downloads":-1,"filename":"ExceptionMan-0.0.0.dev31-py3-none-any.whl","has_sig":false,"md5_digest":"ca3792cf3510eb9cea84927847f85394","packagetype":"bdist_wheel","python_version":"py3","requires_python":">=3.10","size":8797,"upload_time":"2024-11-05T04:35:55","upload_time_iso_8601":"2024-11-05T04:35:55.283179Z","url":"https://files.pythonhosted.org/packages/e8/75/907dc618f49999608e1c45df140b9c236f7696157fc79eb0b0fb73921e33/ExceptionMan-0.0.0.dev31-py3-none-any.whl","yanked":false,"yanked_reason":null},{"comment_text":"","digests":{"blake2b_256":"a876199d0df31773610edc437ab3c6c9fb316a3f703170040fa6c6a1c9c1f841","md5":"599731b35b63cccf76235803bede92e2","sha256":"eeb8462f81cb5bce4bbbb8f38298f473728954cf3ad236aa00ac4ba62c1ae59e"},"downloads":-1,"filename":"exceptionman-0.0.0.dev31.tar.gz","has_sig":false,"md5_digest":"599731b35b63cccf76235803bede92e2","packagetype":"sdist","python_version":"source","requires_python":">=3.10","size":7916,"upload_time":"2024-11-05T04:35:59","upload_time_iso_8601":"2024-11-05T04:35:59.015460Z","url":"https://files.pythonhosted.org/packages/a8/76/199d0df31773610edc437ab3c6c9fb316a3f703170040fa6c6a1c9c1f841/exceptionman-0.0.0.dev31.tar.gz","yanked":false,"yanked_reason":null}],"vulnerabilities":[]},"0.0.0.dev32":{"info":{"author":null,"author_email":null,"bugtrack_url":null,"classifiers":[],"description_content_type":null,"docs_url":null,"download_url":null,"downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":null,"keywords":null,"license":null,"maintainer":null,"maintainer_email":null,"name":"ExceptionMan","package_url":"https://pypi.org/project/ExceptionMan/","platform":null,"project_url":"https://pypi.org/project/ExceptionMan/","project_urls":null,"provides_extra":null,"release_url":"https://pypi.org/project/ExceptionMan/0.0.0.dev32/","requires_dist":["MDit==0.0.0.dev32","rich>=13.5","pygments"],"requires_python":">=3.10","summary":null,"version":"0.0.0.dev32","yanked":false,"yanked_reason":null},"last_serial":25913582,"urls":[{"comment_text":"","digests":{"blake2b_256":"cd072330020c04c040f777e04c11b16432b6500b14e15b9c8001324ce929785c","md5":"73b3e0904c173a8ed9ae29bcf9fb0173","sha256":"7542510f6718c9a0ff5adb4a623cce2f3db9e188cddf9603c1f029f62049aa0b"},"downloads":-1,"filename":"ExceptionMan-0.0.0.dev32-py3-none-any.whl","has_sig":false,"md5_digest":"73b3e0904c173a8ed9ae29bcf9fb0173","packagetype":"bdist_wheel","python_version":"py3","requires_python":">=3.10","size":8795,"upload_time":"2024-11-10T04:00:18","upload_time_iso_8601":"2024-11-10T04:00:18.731225Z","url":"https://files.pythonhosted.org/packages/cd/07/2330020c04c040f777e04c11b16432b6500b14e15b9c8001324ce929785c/ExceptionMan-0.0.0.dev32-py3-none-any.whl","yanked":false,"yanked_reason":null},{"comment_text":"","digests":{"blake2b_256":"bb1cbe28f07bf10a745091ceb5af7651941d9fd540c51c7cf312759947029623","md5":"679c0ef2bf22eb935fd2c9798e06abf3","sha256":"caceb7d955286bf1d3a3d59b1f760e7e0c34b674838a39c93eacdf449d0a7a91"},"downloads":-1,"filename":"exceptionman-0.0.0.dev32.tar.gz","has_sig":false,"md5_digest":"679c0ef2bf22eb935fd2c9798e06abf3","packagetype":"sdist","python_version":"source","requires_python":">=3.10","size":7912,"upload_time":"2024-11-10T04:00:22","upload_time_iso_8601":"2024-11-10T04:00:22.838195Z","url":"https://files.pythonhosted.org/packages/bb/1c/be28f07bf10a745091ceb5af7651941d9fd540c51c7cf312759947029623/exceptionman-0.0.0.dev32.tar.gz","yanked":false,"yanked_reason":null}],"vulnerabilities":[]},"0.0.0.dev4":{"info":{"author":null,"author_email":null,"bugtrack_url":null,"classifiers":[],"description_content_type":null,"docs_url":null,"download_url":null,"downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":null,"keywords":null,"license":null,"maintainer":null,"maintainer_email":null,"name":"ExceptionMan","package_url":"https://pypi.org/project/ExceptionMan/","platform":null,"project_url":"https://pypi.org/project/ExceptionMan/","project_urls":null,"provides_extra":null,"release_url":"https://pypi.org/project/ExceptionMan/0.0.0.dev4/","requires_dist":["mdit==0.0.0.dev3","rich","pygments"],"requires_python":">=3.10","summary":null,"version":"0.0.0.dev4","yanked":false,"yanked_reason":null},"last_serial":25913582,"urls":[{"comment_text":"","digests":{"blake2b_256":"c38807f6e7c22d25ef20fed03a77632eee991a72b0e0f74ed298f739f84969c5","md5":"b4a578bf6cf073f47a7f952b8b0f7d7a","sha256":"c2f2bf8f42d8ab828ef4edc474234db89e70a0b34c3bec2a994a55e6e9dde642"},"downloads":-1,"filename":"ExceptionMan-0.0.0.dev4-py3-none-any.whl","has_sig":false,"md5_digest":"b4a578bf6cf073f47a7f952b8b0f7d7a","packagetype":"bdist_wheel","python_version":"py3","requires_python":">=3.10","size":9027,"upload_time":"2024-09-21T13:30:39","upload_time_iso_8601":"2024-09-21T13:30:39.980288Z","url":"https://files.pythonhosted.org/packages/c3/88/07f6e7c22d25ef20fed03a77632eee991a72b0e0f74ed298f739f84969c5/ExceptionMan-0.0.0.dev4-py3-none-any.whl","yanked":false,"yanked_reason":null},{"comment_text":"","digests":{"blake2b_256":"5c349b466cae3b47665f94513c013c3134fa248b13ed1f35a2a7d2f0091e58bc","md5":"6238d07cb938a22086104c9c623b527b","sha256":"d1028d3ef1ecb8a6682a8befd101041ddb63dab2958464b195ff457aed29b8f3"},"downloads":-1,"filename":"exceptionman-0.0.0.dev4.tar.gz","has_sig":false,"md5_digest":"6238d07cb938a22086104c9c623b527b","packagetype":"sdist","python_version":"source","requires_python":">=3.10","size":8104,"upload_time":"2024-09-21T13:30:41","upload_time_iso_8601":"2024-09-21T13:30:41.363872Z","url":"https://files.pythonhosted.org/packages/5c/34/9b466cae3b47665f94513c013c3134fa248b13ed1f35a2a7d2f0091e58bc/exceptionman-0.0.0.dev4.tar.gz","yanked":false,"yanked_reason":null}],"vulnerabilities":[]},"0.0.0.dev5":{"info":{"author":null,"author_email":null,"bugtrack_url":null,"classifiers":[],"description_content_type":null,"docs_url":null,"download_url":null,"downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":null,"keywords":null,"license":null,"maintainer":null,"maintainer_email":null,"name":"ExceptionMan","package_url":"https://pypi.org/project/ExceptionMan/","platform":null,"project_url":"https://pypi.org/project/ExceptionMan/","project_urls":null,"provides_extra":null,"release_url":"https://pypi.org/project/ExceptionMan/0.0.0.dev5/","requires_dist":["mdit==0.0.0.dev5","rich>=13.5","pygments"],"requires_python":">=3.10","summary":null,"version":"0.0.0.dev5","yanked":false,"yanked_reason":null},"last_serial":25913582,"urls":[{"comment_text":"","digests":{"blake2b_256":"61e6ec4f141013d598b8d70ddc23a419fb3d727e3a87353e7ebe0c4b4804805a","md5":"83224ec17f0333c4d23742ae8df275b9","sha256":"a0d289461a88ab09d49dbfeafb7bde8446ee2fb95663eb5c18f4d7d82f2b11ef"},"downloads":-1,"filename":"ExceptionMan-0.0.0.dev5-py3-none-any.whl","has_sig":false,"md5_digest":"83224ec17f0333c4d23742ae8df275b9","packagetype":"bdist_wheel","python_version":"py3","requires_python":">=3.10","size":9027,"upload_time":"2024-09-22T17:29:39","upload_time_iso_8601":"2024-09-22T17:29:39.221208Z","url":"https://files.pythonhosted.org/packages/61/e6/ec4f141013d598b8d70ddc23a419fb3d727e3a87353e7ebe0c4b4804805a/ExceptionMan-0.0.0.dev5-py3-none-any.whl","yanked":false,"yanked_reason":null},{"comment_text":"","digests":{"blake2b_256":"0bd755b40e07084ffbce97046b2ebcd86cad2fc021b7c1bb988410da9b269585","md5":"8fee2dddd5061de1aad80951c8c4d6d8","sha256":"8e7e9a85b3f489d199296dccfbab6013e1fdb1804da8da8e7f4ec9808bb68f9e"},"downloads":-1,"filename":"exceptionman-0.0.0.dev5.tar.gz","has_sig":false,"md5_digest":"8fee2dddd5061de1aad80951c8c4d6d8","packagetype":"sdist","python_version":"source","requires_python":">=3.10","size":8132,"upload_time":"2024-09-22T17:29:40","upload_time_iso_8601":"2024-09-22T17:29:40.375255Z","url":"https://files.pythonhosted.org/packages/0b/d7/55b40e07084ffbce97046b2ebcd86cad2fc021b7c1bb988410da9b269585/exceptionman-0.0.0.dev5.tar.gz","yanked":false,"yanked_reason":null}],"vulnerabilities":[]},"0.0.0.dev6":{"info":{"author":null,"author_email":null,"bugtrack_url":null,"classifiers":[],"description_content_type":null,"docs_url":null,"download_url":null,"downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":null,"keywords":null,"license":null,"maintainer":null,"maintainer_email":null,"name":"ExceptionMan","package_url":"https://pypi.org/project/ExceptionMan/","platform":null,"project_url":"https://pypi.org/project/ExceptionMan/","project_urls":null,"provides_extra":null,"release_url":"https://pypi.org/project/ExceptionMan/0.0.0.dev6/","requires_dist":["MDit==0.0.0.dev6","rich>=13.5","pygments"],"requires_python":">=3.10","summary":null,"version":"0.0.0.dev6","yanked":false,"yanked_reason":null},"last_serial":25913582,"urls":[{"comment_text":"","digests":{"blake2b_256":"02d6698c3364c02e010aa541343b12104864ac8507df956b3a90cd6e6a000082","md5":"c07c712b0c63688590b13162fc4789f0","sha256":"e5705c68cb5c85a86f8bf314b49c217fd2ce900ab885619036b405774a1ad3b2"},"downloads":-1,"filename":"ExceptionMan-0.0.0.dev6-py3-none-any.whl","has_sig":false,"md5_digest":"c07c712b0c63688590b13162fc4789f0","packagetype":"bdist_wheel","python_version":"py3","requires_python":">=3.10","size":9029,"upload_time":"2024-09-23T12:26:53","upload_time_iso_8601":"2024-09-23T12:26:53.499698Z","url":"https://files.pythonhosted.org/packages/02/d6/698c3364c02e010aa541343b12104864ac8507df956b3a90cd6e6a000082/ExceptionMan-0.0.0.dev6-py3-none-any.whl","yanked":false,"yanked_reason":null},{"comment_text":"","digests":{"blake2b_256":"43c58cd29f4ef8bcf3c6fc9e70fdd8ae6b8985da6c79a15df9f6b7c22a15e81c","md5":"2783203b7c05490b40ffc552d19ef86e","sha256":"2b42c19e679be60711e557932dbf77ee077273c6f7ac91b30f5f706e175e5eb6"},"downloads":-1,"filename":"exceptionman-0.0.0.dev6.tar.gz","has_sig":false,"md5_digest":"2783203b7c05490b40ffc552d19ef86e","packagetype":"sdist","python_version":"source","requires_python":">=3.10","size":8138,"upload_time":"2024-09-23T12:26:54","upload_time_iso_8601":"2024-09-23T12:26:54.646809Z","url":"https://files.pythonhosted.org/packages/43/c5/8cd29f4ef8bcf3c6fc9e70fdd8ae6b8985da6c79a15df9f6b7c22a15e81c/exceptionman-0.0.0.dev6.tar.gz","yanked":false,"yanked_reason":null}],"vulnerabilities":[]},"0.0.0.dev7":{"info":{"author":null,"author_email":null,"bugtrack_url":null,"classifiers":[],"description_content_type":null,"docs_url":null,"download_url":null,"downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":null,"keywords":null,"license":null,"maintainer":null,"maintainer_email":null,"name":"ExceptionMan","package_url":"https://pypi.org/project/ExceptionMan/","platform":null,"project_url":"https://pypi.org/project/ExceptionMan/","project_urls":null,"provides_extra":null,"release_url":"https://pypi.org/project/ExceptionMan/0.0.0.dev7/","requires_dist":["MDit==0.0.0.dev7","rich>=13.5","pygments"],"requires_python":">=3.10","summary":null,"version":"0.0.0.dev7","yanked":false,"yanked_reason":null},"last_serial":25913582,"urls":[{"comment_text":"","digests":{"blake2b_256":"2c9a3fa6ecaf8f90a1188adbe320c0ba2d40ac8b2ed22bfb29e140dbf8f7ce63","md5":"959ff7d00b3fccc317bf82a8c99cf2f8","sha256":"0f7be4f1cb704e3317a1194edc284ac410e5da16ac81d1281385b13457676bab"},"downloads":-1,"filename":"ExceptionMan-0.0.0.dev7-py3-none-any.whl","has_sig":false,"md5_digest":"959ff7d00b3fccc317bf82a8c99cf2f8","packagetype":"bdist_wheel","python_version":"py3","requires_python":">=3.10","size":9028,"upload_time":"2024-09-23T12:35:59","upload_time_iso_8601":"2024-09-23T12:35:59.481863Z","url":"https://files.pythonhosted.org/packages/2c/9a/3fa6ecaf8f90a1188adbe320c0ba2d40ac8b2ed22bfb29e140dbf8f7ce63/ExceptionMan-0.0.0.dev7-py3-none-any.whl","yanked":false,"yanked_reason":null},{"comment_text":"","digests":{"blake2b_256":"1d355d4cdfecc64bac1262ffa60ab8d9e6fb26bde53d43fef62c6ae2a8c61f9b","md5":"21ee444f450d25cdb737915abef79158","sha256":"11337ccb41900a64ea6f1d471da27e82627149ff8dcb421212db6ce08d9596df"},"downloads":-1,"filename":"exceptionman-0.0.0.dev7.tar.gz","has_sig":false,"md5_digest":"21ee444f450d25cdb737915abef79158","packagetype":"sdist","python_version":"source","requires_python":">=3.10","size":8151,"upload_time":"2024-09-23T12:36:00","upload_time_iso_8601":"2024-09-23T12:36:00.320187Z","url":"https://files.pythonhosted.org/packages/1d/35/5d4cdfecc64bac1262ffa60ab8d9e6fb26bde53d43fef62c6ae2a8c61f9b/exceptionman-0.0.0.dev7.tar.gz","yanked":false,"yanked_reason":null}],"vulnerabilities":[]},"0.0.0.dev8":{"info":{"author":null,"author_email":null,"bugtrack_url":null,"classifiers":[],"description_content_type":null,"docs_url":null,"download_url":null,"downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":null,"keywords":null,"license":null,"maintainer":null,"maintainer_email":null,"name":"ExceptionMan","package_url":"https://pypi.org/project/ExceptionMan/","platform":null,"project_url":"https://pypi.org/project/ExceptionMan/","project_urls":null,"provides_extra":null,"release_url":"https://pypi.org/project/ExceptionMan/0.0.0.dev8/","requires_dist":["MDit==0.0.0.dev8","rich>=13.5","pygments"],"requires_python":">=3.10","summary":null,"version":"0.0.0.dev8","yanked":false,"yanked_reason":null},"last_serial":25913582,"urls":[{"comment_text":"","digests":{"blake2b_256":"595ae7cf8a9a3d654b0d2c1cf23a7ad57e4ac1ebe49e38a6c4ce4b65b496cb1d","md5":"41d1ae780006589c4b10bffca667c2cd","sha256":"7baf7c5672c32597289a6ddfac3be3599cd203b7e0b0e440d9051a39cbe936b3"},"downloads":-1,"filename":"ExceptionMan-0.0.0.dev8-py3-none-any.whl","has_sig":false,"md5_digest":"41d1ae780006589c4b10bffca667c2cd","packagetype":"bdist_wheel","python_version":"py3","requires_python":">=3.10","size":9029,"upload_time":"2024-09-23T13:41:10","upload_time_iso_8601":"2024-09-23T13:41:10.469390Z","url":"https://files.pythonhosted.org/packages/59/5a/e7cf8a9a3d654b0d2c1cf23a7ad57e4ac1ebe49e38a6c4ce4b65b496cb1d/ExceptionMan-0.0.0.dev8-py3-none-any.whl","yanked":false,"yanked_reason":null},{"comment_text":"","digests":{"blake2b_256":"62ff610b9e8f3d658e47c6d4a0fe897912e1ce6e5de65849f48927e55f9a6344","md5":"277e47afb274d25f1c0ce1e33eb09208","sha256":"02602b08c98396ff6c3149c738cda1fc9e03094cf69bf272935d34a928bb422f"},"downloads":-1,"filename":"exceptionman-0.0.0.dev8.tar.gz","has_sig":false,"md5_digest":"277e47afb274d25f1c0ce1e33eb09208","packagetype":"sdist","python_version":"source","requires_python":">=3.10","size":8149,"upload_time":"2024-09-23T13:41:11","upload_time_iso_8601":"2024-09-23T13:41:11.543895Z","url":"https://files.pythonhosted.org/packages/62/ff/610b9e8f3d658e47c6d4a0fe897912e1ce6e5de65849f48927e55f9a6344/exceptionman-0.0.0.dev8.tar.gz","yanked":false,"yanked_reason":null}],"vulnerabilities":[]},"0.0.0.dev9":{"info":{"author":null,"author_email":null,"bugtrack_url":null,"classifiers":[],"description_content_type":null,"docs_url":null,"download_url":null,"downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":null,"keywords":null,"license":null,"maintainer":null,"maintainer_email":null,"name":"ExceptionMan","package_url":"https://pypi.org/project/ExceptionMan/","platform":null,"project_url":"https://pypi.org/project/ExceptionMan/","project_urls":null,"provides_extra":null,"release_url":"https://pypi.org/project/ExceptionMan/0.0.0.dev9/","requires_dist":["MDit==0.0.0.dev9","rich>=13.5","pygments"],"requires_python":">=3.10","summary":null,"version":"0.0.0.dev9","yanked":false,"yanked_reason":null},"last_serial":25913582,"urls":[{"comment_text":"","digests":{"blake2b_256":"b24e4b9d1cbc509b39e2d3dd14a93d497177f3f8a63d1f5fada29d61bf63371c","md5":"fa0447984811595f22002ffc54a33de4","sha256":"82f2405d2ddb0c3c9cbd5f285c2782eff531e9f813371d64014128aabda08ab2"},"downloads":-1,"filename":"ExceptionMan-0.0.0.dev9-py3-none-any.whl","has_sig":false,"md5_digest":"fa0447984811595f22002ffc54a33de4","packagetype":"bdist_wheel","python_version":"py3","requires_python":">=3.10","size":9030,"upload_time":"2024-09-23T18:50:47","upload_time_iso_8601":"2024-09-23T18:50:47.509249Z","url":"https://files.pythonhosted.org/packages/b2/4e/4b9d1cbc509b39e2d3dd14a93d497177f3f8a63d1f5fada29d61bf63371c/ExceptionMan-0.0.0.dev9-py3-none-any.whl","yanked":false,"yanked_reason":null},{"comment_text":"","digests":{"blake2b_256":"af63e4eee104554828cdcd1b5276c21f9f8735dc90bba642d18ddc98a5a067ff","md5":"f20b5fcf84c806d8c8b0c3b0352f3516","sha256":"14a1848a2925a91aeafb55a97bc26c521e4db9f4aad2f4b53b7b18eaf4a29314"},"downloads":-1,"filename":"exceptionman-0.0.0.dev9.tar.gz","has_sig":false,"md5_digest":"f20b5fcf84c806d8c8b0c3b0352f3516","packagetype":"sdist","python_version":"source","requires_python":">=3.10","size":8135,"upload_time":"2024-09-23T18:50:50","upload_time_iso_8601":"2024-09-23T18:50:50.006828Z","url":"https://files.pythonhosted.org/packages/af/63/e4eee104554828cdcd1b5276c21f9f8735dc90bba642d18ddc98a5a067ff/exceptionman-0.0.0.dev9.tar.gz","yanked":false,"yanked_reason":null}],"vulnerabilities":[]}}